net/node/node.go

```diff
@@ -381,6 +381,9 @@ func (node *node) StoreFlightHeight(height uint32) {
 func (node *node) GetFlightHeightCnt() int {
 	return len(node.flightHeights)
 }
+func (node *node) GetFlightHeights() []uint32 {
+	return node.flightHeights
```

Comment:
Slice "node.flightHeights" was used  concurrently in different go routine, should add sync to this slice.